packages/client/src/api/user/models.ts

```diff
@@ -0,0 +1,12 @@
+import { TUser } from 'src/api/auth/models';
+
+export type TUserData = Omit<TUser, 'password'> & UserDisplayName;
+
+type UserDisplayName = {
+  display_name: 'string';
+};
```

Comment:
Гуд, но как будто бы пока можно сделать даже так
```suggestion
export type TUserData = Omit<TUser, 'password'> & { display_name: string };
```